File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        if (x<0)
            return false;
        int rever=0,rem,n=x;
        while(x!=0){
            rem=x%10;
            rever=rever*10+rem;
            x/=10;
        }
        return n==rever;
    }
}

File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        int y=x;
        long int palin=0;
        int rem;
        while(x!=0) {
            rem=x%10;
            palin = palin*10 +rem;
            x=x/10;
        }
        if(y<0)
        return false;
        else if(y==palin)
        return true;
        else{
            return false;
        }
    }
};